### ISSUE TITLE
Need a cue to "Move down to rub harder" after some non-productive rubbing

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#125]
Non-visual users have a hard time knowing about the air space between the books on page load and after a break away. They basically get little to no feedback when they move the book back and forth but they are not actually causing friction. We do not want to directly describe the airspace in between the 2 books.

It might work to cue them that moving down to rub harder might be more productive. 

When books are not touching:
- Allow 1 pair of Left-Right alerts and then on the next horizontal key press when books are not touching replace the direction with, “Move down to rub harder.”

Ideally, we should not have to cue the user forever, so provide the cuing alert every second pair left-right key press until they move vertically. 

- If they move up and hit the top, they will get the new alert from #124, "At top. Move down to rub harder." 
- And once they hit the book, they will hear the other new alert from #124 "Down. Rub fast or slow."

If at all possible, follow this pattern 2 times and then go back to the very short left-right alerts. After a reset be sure to provide the alert again.

**Comment by terracoda** [comment]
@zepumph, I hope this is clear. Let me know if yo have any questions.

**Comment by terracoda** [comment]
@zepumph and I discussed this issue, and things seem clear.

We also discussed, silencing the left-right alerts a little more dramatically if books are not touching and the sim is in a cooling state to ensure left-right alerts do not interrupt all the cooling alerts. 

I am opening a separate issue for this idea.

**Comment by zepumph** [comment]
@terracoda I implemented this such that every time there is a pair of "left/right" alerts, the next alert will be "Move down to rub harder." Then it will restart to go back to left/right again. Each time a left/right pair happens, the move down will occur afterwards. I'm not sure if that is exactly what we discussed Friday, sorry I should have taken better notes. Please let me know how this should be tweaked.

**Comment by terracoda** [comment]
@zepumph this working well. 
Once #124 is all fixed up, we can revisit if necessary. I'll go ahead and close this for now.
